Added nfs configuration handling

simo-tuomisto opened the pull request:
Modern versions of nfs-utils do not use `/etc/sysconfig/nfs` anymore. Since 2016 the move has been towards `/etc/nfs.conf` and in CentOS 8 this change has been [completed](https://fedoraproject.org/wiki/Changes/nfs.conf). CentOS 7.7 and newer support `/etc/nfs.conf`, so the old logic is no longer necessary.

simo-tuomisto committed:
Added nfs.conf configuration file management and added molecule tests for the nfs servers

simo-tuomisto committed:
Added .yamllint for molecule tests

mhakala commented:
Just a question. What happens for older servers that already use /etc/sysconfig/nfs. Does /etc/nfs.conf take presense first or should /etc/sysconfig/nfs be removed with the role?

simo-tuomisto commented:
I verified this and it [looks like the old configuration is used](https://bugzilla.redhat.com/show_bug.cgi?id=1418041). I'll update the roles to remove the old configuration if present.

simo-tuomisto committed:
Added default template for /etc/sysconfig/nfs.

This commit adds a default template for /etc/sysconfig/nfs.
This empty template makes certain that there's no conflicting
configurations between old and new configuration methods.

simo-tuomisto committed:
Added backup to /etc/sysconfig/nfs configuration changing.
